pkg/gedis/commands_provision.go

```diff
@@ -149,8 +149,8 @@ func (g *Gedis) Feedback(id string, r *provision.Result) error {
 	}
 
 	_, err := g.Send("workload_manager", "set_workload_result", Args{
-		"reservation_id": id,
-		"result":         result,
+		"global_workload_id": id,
```

Comment:
This is actually part of the Feedback "Result"  error. but i got so distracted by the schema error in BCDB (it took most of the day) that i totally forgot I changed that as well.